test/testdrive/introspection-sources.td

```diff
@@ -265,8 +265,8 @@ $ set-regex match=\d{13} replacement=<TIMESTAMP>
 > CREATE DEFAULT INDEX ii_arr ON vv_arr
 
 # Under arrangement specialization of only empty values, a single-column relation
-# should demand 64 bytes per tuple instead of 96.
-> SELECT records >= 300, size >= 300*64, capacity >= 300*64 FROM mz_internal.mz_dataflow_arrangement_sizes WHERE name LIKE '%ii_arr' OR name LIKE '%mv_arr'
+# should demand 48 bytes per tuple instead of 96.
+> SELECT records >= 300, size >= 300*48, capacity >= 300*48 FROM mz_internal.mz_dataflow_arrangement_sizes WHERE name LIKE '%ii_arr' OR name LIKE '%mv_arr'
```

Comment:
```suggestion
# should demand 56 bytes per tuple instead of 96.
> SELECT records >= 300, size >= 300*56, capacity >= 300*56 FROM mz_internal.mz_dataflow_arrangement_sizes WHERE name LIKE '%ii_arr' OR name LIKE '%mv_arr'
```

The empty-value specialization stores keys of `Row` and `usize`, values of `()` and `usize`, and `(T, R)` of `(usize, usize)`, which should sum up to 56 bytes.

Reply:
I think after https://github.com/MaterializeInc/materialize/pull/23033 these are now the right values? Let me know and I can update them!